src/protocols/tacacs/encode.c

```diff
@@ -246,7 +261,6 @@ ssize_t fr_tacacs_encode(uint8_t *buffer, size_t buffer_len, uint8_t const *orig
 			/*
 			 *	Encode 4 mandatory fields.
 			 */
-			p = packet->authen.start.body;
 			ENCODE_FIELD_STRING8(packet->authen.start.user_len, attr_tacacs_user_name);
```

Comment:
These should be swapped out for DBUFF variables which do the buffer advancement instead of using `FR_DBUFF_ADVANCE_RETURN`